src/DrupalOrg/VersionGroup.php

```diff
@@ -45,7 +45,7 @@ public function getNextVersion(string $currentVersion): string
         $nextVersions = array_filter(
             $normalizedVersions,
             function (string $version) use ($normalizedCurrentVersion): bool {
-                return Comparator::greaterThanOrEqualTo($version, $normalizedCurrentVersion);
+                return Comparator::greaterThan($version, $normalizedCurrentVersion);
```

Comment:
I do not agree with this change.

`getNextVersion()` is called from `ProjectFetcher->getSecureVersion()`. The purpose of this function is to get the secure version of a project given it's current version. `ProjectFetcher->getSecureVersion()` creates a version group containing the the secure versions of a project and then calls `getNextVersion()` with the current version of the project.

If the current version is secure then it will be included in the version group and `getNextVersion()` should return the current version. Comparing using `greaterThanOrEqualTo()` is important here.

If the current version is not secure then it will not be included in the version group and the next secure version should be returned. Using `greaterThanOrEqualTo()` will get us there as well.